rubin_jupyter_utils/hub/scanrepo/scanrepo.py

```diff
@@ -489,17 +475,13 @@ def _reduce_results(self):
                 vname = res["name"]
                 reduced_results[vname] = {
                     "name": vname,
-                    "id": res["id"],
```

Comment:
I don't think we can throw away the id (assuming I remember correctly and that's the hash of the image).  The reason is a little subtle and easy to miss.  If the "recommended" image is NOT otherwise in the current displayed list (let's say the weeklies have failed SimonCI for several weeks), it *still* needs to be prepulled, and it is very helpful if the spawned image contains the actual image name.

Right now it appears that red-five has not been prepulling "recommended."  First spawn timed out, second one is taking a really long time but (presumably) is pulling the image.

Most people are going to choose the recommended image.  We cannot make them wait 15 minutes for it to pull just because it's drifted outside the range of the usually-pulled weekly.

Reply:
Sounded like you knew why this was happening?

Reply:
We can throw away the ID no problem, and recommended still works.  That seems to be a red herring.  The issue with being able to start was a completely different thing with adding the host, (you can't add hub.docker.com to an image, it has to be docker.io I guess)